src/Tizen.NUI/src/public/Theme/Theme.cs

```diff
@@ -107,6 +107,24 @@ public Theme(string xamlFile) : this()
         /// <since_tizen> 9 </since_tizen>
         public string Version { get; set; } = null;
 
+        /// <summary>
+        /// The url of small broken image
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string SmallBrokenImageUrl { get; set; } = null;
+
+        /// <summary>
+        /// The url of  broken image
```

Comment:
check space of summary. 
```suggestion
        /// The url of broken image
```